tools/extras/install_morfessor.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Copyright  2017  Atlas Guide (Author : Lucas Jo)
+#
+# Apache 2.0
+#
+
+echo "#### installing morfessor"
+dirname=morfessor
+if [ ! -d ./$dirname ]; then
+  mkdir -p ./$dirname
+  git clone https://github.com/aalto-speech/morfessor.git morfessor ||
+    {
+      if [ $? -ne 0 ]; then
+        echo  >&2 "$0: Error git clone operation "
+        echo  >&2 "  Failed in cloning the github repository (https://github.com/aalto-speech/morfessor.git)"
+        echo  >&2 "  Download morfessor-2.0.1.tar.gz instead"
+        wget http://morpho.aalto.fi/projects/morpho/morfessor-2.0.1.tar.gz
+        tar -zxvf morfessor-2.0.1.tar.gz
+        mv Morfessor-2.0.1/ $dirname
+      fi
+    }
+fi
+
+# local installation
+site_packages_dir=$(python -m site --user-site | grep -oE "lib.*")
+wd=`pwd`
+wd=`readlink -f $wd || pwd`
+export MORFESSOR="$wd/morfessor"
+export PYTHONPATH="${PYTHONPATH:-}:$MORFESSOR/${site_packages_dir}"
```

Comment:
I still think this is too complex for no clear reason (reason clear to me) -- you could just install into $MORFESSOR/local/ (or whatever) and this would greatly simplify the script.

Reply:
@jtrmal yep I agree on it. 
Anyway it seems like @psmit shows us more clear way. 
After scripts in scripts/ has been changed to executables, I'll update this code in more simpler way.